src/task01.java

```diff
@@ -0,0 +1,93 @@
+
+interface control {
+
+    void speed();
+}
+
+abstract class Vehicle implements control{
+
+    String type;
+    public Vehicle(String typ)
+    {
+        this.type = typ;
+    }
+    abstract void info();
+}
+
+class Road extends Vehicle{
+
+    public Road()
+    {
+        super("Road");
+    }
+
+    static String owner_name, model_no;
+
+    public void info()
+    {
+        owner_name = "Ben";
+        model_no = "6tJ4690";
+    }
+    static int sp;
+    public void speed()
+    {
+        sp = 80;
+    }
+}
+
+class Air extends Vehicle{
+
+    public Air()
+    {
+        super("Air");
+    }
+
+    static String owner_name, model_no;
+
+    public void info()
+    {
+        owner_name = "Jack";
+        model_no = "4t678";
+    }
+    static int sp;
+    public void speed()
+    {
+        sp =1080;
+    }
+}
+
+class two_wheeler extends Road{
+
+    void display(){
```

Comment:
why `display()` ? We were supposed to display the info via `info()` method not adding display() method.            Hint: override the info() method at proper places